internal/core/application/indexer.go

```diff
@@ -266,16 +272,18 @@ func (i *indexerService) GetVtxoChain(
 
 					// populate newNextVtxos with checkpoints inputs
 					for _, in := range ptx.UnsignedTx.TxIn {
-						newNextVtxos = append(newNextVtxos, domain.Outpoint{
-							Txid: in.PreviousOutPoint.Hash.String(),
-							VOut: in.PreviousOutPoint.Index,
-						})
+						childKey := fmt.Sprintf("%s:%d", in.PreviousOutPoint.Hash.String(), in.PreviousOutPoint.Index)
```

Comment:
```suggestion
						childKey := in.PreviousOutPoint.String()
```
I think also this type encode string as `hash:vout`

Reply:
Done